src/ui/buttons/buttons.jsx

```diff
@@ -150,6 +152,47 @@ export function validateButtonProps(props: ButtonPropsInputs) {
   normalizeButtonProps(props);
 }
 
+function calculateMsgPosition({
+  message,
+  tagline,
+  layout,
+  fundingSources,
+  fundingSource,
+}): string {
+  if (!message) {
+    return null;
+  }
+  const { position } = message;
+  // Cases for throwing an error
+  if (tagline && layout === BUTTON_LAYOUT.HORIZONTAL && !fundingSource) {
```

Comment:
Would it be worthwhile to turn this into a variable such as `showTagline` in the `Buttons` function that could be passed into this function so that `showTagline` could be reused also where the `<Tagline />` component is being considered. This could help introducing an accidental bug in the future if the conditional here to determine whether or not to display the tagline is updated in one place but not the other.

Reply:
very good call